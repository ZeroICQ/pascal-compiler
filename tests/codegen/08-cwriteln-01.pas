var
    mychar : char;

begin
    mychar := 'h';
    cwriteln(mychar);
    cwriteln('e');
    cwriteln('l');
    cwriteln('l');
    cwriteln('o');
    cwriteln('!');
    cwriteln('9');
end.