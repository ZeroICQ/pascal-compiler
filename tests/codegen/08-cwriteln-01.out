h
e
l
l
o
!
9